catalyst/contrib/utils/__init__.py

```diff
@@ -14,17 +16,27 @@
     calculate_confusion_matrix_from_tensors,
 )
 from .dataset import create_dataset, split_dataset_train_test, create_dataframe
-from .image import (
-    has_image_extension,
-    imread,
-    imwrite,
-    imsave,
-    mask_to_overlay_image,
-    mimread,
-    mimwrite_with_meta,
-    tensor_from_rgb_image,
-    tensor_to_ndimage,
-)
+
+try:
+    from .image import (
+        has_image_extension,
+        imread,
+        imwrite,
+        imsave,
+        mask_to_overlay_image,
+        mimread,
+        mimwrite_with_meta,
+        tensor_from_rgb_image,
+        tensor_to_ndimage,
+    )
+except ImportError as ex:
```

Comment:
btw, what do you think about changing it to `(ImportError, NameError, AttributeError)`?